Gek62/hometask-5/5.4.3.sql

```diff
@@ -0,0 +1,3 @@
+SELECT CONCAT(last_name, '-', first_name) AS full_name
+FROM actor
+WHERE first_name LIKE 'w%' OR last_name LIKE '%h'
```

Comment:
Unfortunately, this is still not case-insensitive, it works with lower case only.
Consider using Postgres' `ILIKE` operator or `LOWER()`.